src/main/java/com/manardenza/entity/Reservation.java

```diff
@@ -7,9 +7,12 @@
 
 @Data
 @AllArgsConstructor
-public class Reservation {
+public class Reservation extends AbstractObject {
 
     private Date reservedFrom;
     private Date reservedTo;
     private User reservedUser;
+    private Room reservedRoom;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "reservedRoom" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)